Hyperbolic_triangulation_2/demo/Hyperbolic_triangulation_2/include/internal/Qt/HyperbolicVoronoiGraphicsItem.h

```diff
@@ -99,23 +99,13 @@ VoronoiGraphicsItem<DT>::paint(QPainter *painter, const QStyleOptionGraphicsItem
   
   painter->setPen(edgesPen());
   
-  // delete
-  QPen temp = painter->pen();
-  QPen old = temp;
-  temp.setWidthF(0.01);
-  painter->setPen(temp);
-  //
-  
   for(typename DT::All_edges_iterator eit = dt->all_edges_begin();
       eit != dt->all_edges_end();
       eit++) 
     {
       typename DT::Hyperbolic_segment s = dt->dual(*eit);
       pos << s;
     } 
- 
-  // delete
-  painter->setPen(old);
```

Comment:
This should be OK, it's a part of the demo and it has no critical impact on the actual code.